Pds4File/pds4_rules/cassini_vims.py

```diff
@@ -0,0 +1,763 @@
+####################################################################################################################################
+# rules/COISS_xxxx.py
+####################################################################################################################################
```

Comment:
Update the comment, same for other rules under Pds4File/pds4_rules/

Reply:
Done